website/docs/r/dashboard.html.markdown

```diff
@@ -131,7 +131,7 @@ All nested `widget` blocks support the following common arguments:
 
 <a name="cross-account-widget-help"></a>
 
- -> **Configurating cross-account widgets:** To configure a cross-account widget with an account different from the account associated with your API key, you must set the widget's `account_id` attribute to the account ID you wish to pull data from. Also note, the provider must be configured with an API Key that is scoped to a user with proper permissions to access and perform operations in other accounts that fall within or under the account associated with your API key. To facilitate cross-account widgets, we recommend [configuring the provider with a Personal API Key](/docs/providers/newrelic/guides/provider_configuration.html#configuration-via-the-provider-block) from a user with **admin permissions** and access to the subaccount you would like to display data for in the widget.
+ -> **Configurating cross-account widgets:** To configure a cross-account widget with an account different from the account associated with your API key, you must set the widget's `account_id` attribute to the account ID you wish to pull data from. Also note, the provider must be configured with an API Key that is scoped to a user with proper permissions to access and perform operations in other accounts that fall within or under the account associated with your API key. To facilitate cross-account widgets, we recommend [configuring the provider with a Personal API Key](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/guides/provider_configuration#configuration-via-the-provider-block) from a user with **admin permissions** and access to the subaccount you would like to display data for in the widget.
```

Comment:
Nit: s/configurating/configuring/

Also you can use a relative path here if you like: `../guides/provider_configuration`

Reply:
I tried the relative paths thing and it worked locally in the old docs stuff, but didn't work for the new registry docs site, but I probably just have the paths off. 

Also "configurating" isn't a word???? 🤣